README.md

```diff
@@ -60,6 +60,9 @@ The `*_test.toml` files contain the requests that will be made.  They're execute
 	method="GET"
 	url="http://{{.base_service_url}}/get"
 	want_code=200
+[litmus.test.query]
```

Comment:
Of course we could just add the query params to the `url` and it'd work pretty well. But I feel that explicitly defining them like this is more readable and testable

Reply:
And we can still use query params for the die-hards, so yep, anything that can make the tool easier to use the better it’ll be.